python/MooseDocs/extensions/content.py

```diff
@@ -211,6 +211,8 @@ def createToken(self, parent, info, page):
             msg = "The 'destination' setting is required for the !content next and !content previous commands."
             raise exceptions.MooseDocsException(msg)
 
+        p_node = self.translator.findPage(self.settings['destination'])
+        page['dependencies'].add(p_node.uid)
```

Comment:
@aeslaughter I don't understand why we need this, because I noticed no difference between with and without it. What does it actually do?

Reply:
I figured out that this attribute is necessary for the Include Extension, but it does nothing here, so I will delete it. It also produces a key error in python unit tests, but not during a live serve, which is unlike the key errors we were dealing with before. This will be encountered when/if a unit test is create for Include, so I guess we need not worry about it for now.

Reply:
It does do something. During the livereload if you change a heading in a page used in the outline, the outline will also update.